### PR TITLE
Add some more German Translations

### PR DESCRIPTION
**Pull request by MTRNord**
Added some missing translations

### CODE REVIEW

File: resources/langs/nheko_de.ts [hunk 1]
```diff
@@ -6,89 +6,89 @@
...
         <location filename="../../src/dialogs/CreateRoom.cc" line="43"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>Titel</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="46"/>
         <source>Topic</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Thema/Beschreibung</translation>
```
Comment (MTRNord) [inline resources/langs/nheko_de.ts]: This needs to be discussed which one fits best in here. I guess @krombel might have some ideas on this

File: resources/langs/nheko_de.ts [hunk 2]
```diff
@@ -6,89 +6,89 @@
...
         <location filename="../../src/dialogs/LeaveRoom.cc" line="29"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/LeaveRoom.cc" line="39"/>
         <source>Are you sure you want to leave?</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (krombel) [inline resources/langs/nheko_de.ts]: Translation missing.
I would suggest "Bist du sicher, dass du den Raum verlassen willst?"

File: resources/langs/nheko_de.ts [hunk 3]
```diff
@@ -139,58 +139,60 @@
...
+        <translation>Löse das reCAPTCHA und drücke den &quot;Bestätigen&quot;-Knopf</translation>
     </message>
 </context>
 <context>
     <name>ReadReceipts</name>
     <message>
         <location filename="../../src/dialogs/ReadReceipts.cc" line="92"/>
         <source>Read receipts</source>
-        <translation type="unfinished"></translation>
+        <translation>Lese Bestätigungen</translation>
```
Comment (krombel) [inline resources/langs/nheko_de.ts]: I would prefer to have just one word

File: resources/langs/nheko_de.ts (hunk 1 above)
Comment (krombel) [reply resources/langs/nheko_de.ts]: In Riot this is called "Thema". And that would be a more direct translation.
"Beschreibung" is "description". As this field is mostly used for that I could agree with that translation but I would prefer to have that consistent to "upstream"

File: resources/langs/nheko_de.ts [hunk 4]
```diff
@@ -6,89 +6,89 @@
...
         <location filename="../../src/dialogs/CreateRoom.cc" line="57"/>
         <source>Room Visibility</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Sichtbarkeit</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="66"/>
         <source>Room Preset</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Vorlage</translation>
```
Comment (krombel) [inline resources/langs/nheko_de.ts]: I would prefer a single word - or to connect those two with "-"

File: resources/langs/nheko_de.ts [hunk 5]
```diff
@@ -6,89 +6,89 @@
...
         <location filename="../../src/dialogs/CreateRoom.cc" line="49"/>
         <source>Alias</source>
-        <translation type="unfinished"></translation>
+        <translation>Alias</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="57"/>
         <source>Room Visibility</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Sichtbarkeit</translation>
```
Comment (krombel) [inline resources/langs/nheko_de.ts]: I would prefer a single word - or to connect those two with "-"

**Comment by MTRNord** [comment]
Oh fun dont these translations get applied?

File: resources/langs/nheko_de.ts [hunk 6]
```diff
@@ -139,58 +139,60 @@
...
+        <translation>Abbrechen</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/PreviewUploadOverlay.cc" line="114"/>
         <source>Media type: %1
 Media size: %2
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Medien Typ: %1
+Medien Größe: %2
```
Comment (krombel) [inline resources/langs/nheko_de.ts]: I would prefer to connect "Medien {Typ,Größe}" with "-"

File: resources/langs/nheko_de.ts (hunk 3 above)
Comment (MTRNord) [reply resources/langs/nheko_de.ts]: 👍

File: resources/langs/nheko_de.ts (hunk 4 above)
Comment (MTRNord) [reply resources/langs/nheko_de.ts]: Going for a single word 👍

File: resources/langs/nheko_de.ts (hunk 5 above)
Comment (MTRNord) [reply resources/langs/nheko_de.ts]: Going for a single word

File: resources/langs/nheko_de.ts (hunk 6 above)
Comment (MTRNord) [reply resources/langs/nheko_de.ts]: 👍

File: resources/langs/nheko_de.ts (hunk 2 above)
Comment (MTRNord) [reply resources/langs/nheko_de.ts]: Whoops missed to type it in

**Pull request merged**